hazelcast/src/test/java/com/hazelcast/IOUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.hazelcast;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.function.Function;
+
+public class IOUtils {
```

Comment:
This helper feels to be specific to config, I think it should be under `com.hazelcast.config.helpers`. We have `DeclarativeConfigFileHelper` there for similar reasons. It doesn't use static methods, but I think these methods can go there. Could you please check if we can add them there?